app/views/layouts/myorders.html.erb

```diff
@@ -0,0 +1 @@
+<%= yield %>
```

Comment:
add some basic HTML skeleton here. copy from the application.html.erb and make some changes.